dotnet/src/dotnetframework/GxClasses/Domain/GxHttpClient.cs

```diff
@@ -1363,6 +1384,25 @@ private Encoding ExtractEncodingFromCharset(string responseText, string regExpP,
 
 		public override string ToString()
 		{
+			if (_chunkedResponse && _receiveStream != null)
```

Comment:
Shouldn't we ask if the Response Header "Transfer-Encoding"  == chunked before trying to read as chunked? Even if the Developer told GeneXus that chunkedResponseHandling is enabled?

Maybe _receiveStream  is null in that case, if that the case, ignore this comment.

Reply:
You are correct. The Transfer-Encoding header determines whether the response is chunked (_chunkedResponse is true).

Reply:
_chunkedResponse now is true when response.Headers.TransferEncodingChunked.HasValue & response.Headers.TransferEncodingChunked.Value is true